drivers/gles3/rasterizer_canvas_gles3.cpp

```diff
@@ -1608,6 +1608,8 @@ void fragment() {
 RasterizerCanvasGLES3::~RasterizerCanvasGLES3() {
 	GLES3::MaterialStorage *material_storage = GLES3::MaterialStorage::get_singleton();
 
+	memdelete(state.instance_data_array);
```

Comment:
This is initialized with `memnew_arr` so I'm guessing it probably needs to be freed with `memdelete_arr`?